utils/rpms/daos.spec

```diff
@@ -380,6 +376,7 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %files tests
 %dir %{_prefix}/lib/daos
 %{_prefix}/lib/daos/TESTING
+%exclude %{_prefix}/lib/daos/TESTING/ftest/list_tests.py
```

Comment:
Having to do this exclude here I think indicates that this PR is not meeting it's goal since this python2 tool would be installed on a source-built installation.